presto-hive/src/main/java/io/prestosql/plugin/hive/HiveColumnHandle.java

```diff
@@ -67,7 +75,13 @@
     public static final HiveType PARTITION_HIVE_TYPE = HIVE_STRING;
     public static final Type PARTITION_TYPE_SIGNATURE = VARCHAR;
 
-    private static final String UPDATE_ROW_ID_COLUMN_NAME = "$shard_row_id";
+    public static final int UPDATE_ROW_ID_COLUMN_INDEX = -16;
+    public static final String UPDATE_ROW_ID_COLUMN_NAME = "row__id";
```

Comment:
No, it is not significant, and the column can't be selected.  It exists solely to convey the type of the triple `(originalTransaction, bucket, rowId)` to the `HiveUpdatablePageSource.deleteRows()`.  There would be no harm in putting $ in the front of the name.  Is that what you would advise?

Reply:
if a table happens to  have a column with such a name, will it cause a name collision?

yes, i'd prefer `$` in the name and maybe a `// not selectable` comment
(and then single underscore instead of double)